REC2Plugin.py

```diff
@@ -107,12 +95,11 @@ def __showSetupDialog(self):
         QgsProject.instance().writeEntry("fishdb", "recLayer", self.recLayer.id())
 
     def __onProjectCreated(self):
-        self.actionGroup.setEnabled(False)
+        self.actionGroup.setEnabled(True)
```

Comment:
You can remove the action group logic completely since now the action is always active.